d2bs/kolbot/libs/common/Misc.js

```diff
@@ -1938,6 +1938,16 @@ MainLoop:
 		}
 
 		return obj;
+	},
+
+	poll: function (testFunc, timeOut = 6000, pollDelay = 40) {
+		let startTime = getTickCount(), ret;
```

Comment:
declare any unassigned variables before assigned variables
leave newline between declaration and while statement
leave newline between return statement and the delay 
leave new line between while statement and return statement